gcd/src/main/java/org/spine3/server/storage/datastore/DsEventStorage.java

```diff
@@ -206,18 +252,20 @@ public boolean apply(@Nullable EventStorageRecord event) {
                 return true;
             }
 
+            boolean atLeastOneNonEmptyFilter = false;
             for (EventFilter filter : eventFilters) {
```

Comment:
Could this `for` be collapsed by using either `Collections2.filter` or `Iterables.filter`? Just to have less lines of code.